content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,169 @@
+be visible. Depending on installed applications and configurations, some fields may not be
+displayed.
+
+.. _recruitment/applicant-details:
+
+Applicant section
+-----------------
+
+- :guilabel:`Subject/Application Name`: this is the only required field. Enter the title for the
+  card. Typically this is the applicant's name and job position being applied to, for example:
+  `John Smith - Experienced Developer`. This field is not visible in the Kanban view, unless the
+  :guilabel:`Applicant's Name` is left blank. If there is no :guilabel:`Applicant's Name`, then the
+  :guilabel:`Subject/Application Name` is what is displayed on the applicant card in the Kanban
+  view.
+- :guilabel:`Applicant's Name`: enter the applicant's name.
+- :guilabel:`Email`: enter the applicant's email address.
+- :guilabel:`Phone`: enter the applicant's phone number.
+- :guilabel:`Mobile`: enter the applicant's mobile number.
+- :guilabel:`LinkedIn Profile`: enter the web address for the applicant's personal profile on
+  LinkedIn.
+- :guilabel:`Degree`: select the applicant's highest level of education earned from the drop-down
+  menu. Options are :guilabel:`Graduate`, :guilabel:`Bachelor Degree`, :guilabel:`Master Degree`, or
+  :guilabel:`Doctoral Degree`. The :guilabel:`Graduate` option indicates graduating the highest
+  level of school before a Bachelor's degree, such as a high school or secondary school diploma,
+  depending on the country.
+- :guilabel:`Interviewers`: using the drop-down menu, select the people who will conduct the
+  interviews. The selected people must have either *recruiter* or *officer* rights configured for
+  the *Recruitment* application to appear in the drop-down list. Refer to the :doc:`Access rights
+  <../../general/users/access_rights>` document for more information.
+- :guilabel:`Recruiter`: select the user responsible for the entire recruitment process for the job
+  position.
+- :guilabel:`Evaluation`: represents a rating for the applicant: one star is :guilabel:`Good`, two
+  stars is :guilabel:`Very Good`, and three stars is :guilabel:`Excellent.`
+- :guilabel:`Source`: using the drop-down menu, select where the applicant learned about the job
+  position. The following options come pre-configured in Odoo: :guilabel:`Search engine`,
+  :guilabel:`Lead Recall`, :guilabel:`Newsletter`, :guilabel:`Facebook`, :guilabel:`Twitter`,
+  :guilabel:`LinkedIn`, :guilabel:`Monster`, :guilabel:`Glassdoor`, and :guilabel:`Craigslist`. To
+  add a new :guilabel:`Source`, type in the source, then click :guilabel:`Create "(new source)"`.
+- :guilabel:`Medium`: using the drop-down menu, specify how the job listing was found. The
+  pre-configured options are: :guilabel:`Banner`, :guilabel:`Direct`, :guilabel:`Email`,
+  :guilabel:`Facebook`, :guilabel:`Google Adwords`, :guilabel:`LinkedIn`, :guilabel:`Phone`,
+  :guilabel:`Television`, :guilabel:`Twitter` (now known as "X"), or :guilabel:`Website`. To add a
```

Comment:
I'm also seeing 'SMS' and 'Website' in this field. Probably best to do a double-check during your revisions to ensure the accuracy of these options...